packages/cypress/README.md

```diff
@@ -1,7 +1,67 @@
-# Cypress commons
+# Cloudify UI Common Cypress
 
-This directory contains resources common to Cypress tests:
+[![npm](https://img.shields.io/npm/v/cloudify-ui-common-cypress.svg?style=flat)](https://www.npmjs.com/package/cloudify-ui-common-cypress)
 
-* `plugins` - plugins setup and runtime configuration logic
-* `support` - custom commands definitions
-* `cypress.json` - base Cypress runner static configuration
+This package contains resources common to [Cypress](https://cypress.io/) tests:
+
+* Cypress plugins setup and runtime configuration logic - see [`plugins` directory](./src/plugins)
+* Cypress custom commands definitions - see [`support` directory](./src/support)
+* Cypress runner static configuration - see [`cypress.json` file](./cypress.json) 
+
+
+## Installation
+
+```npm
+npm install cloudify-ui-common-cypress
+```
+
+## Usage
+
+### Plugins
+
+Cypress plugin and configuration setup can be done the following way:
+
+```typescript
+// test/cypress/plugins/index.ts
+import performCommonSetup from 'cloudify-ui-common-cypress/plugins';
+import getWebpackConfig from '../../../webpack.config';
+
+const setupPluginsAndConfig: Cypress.PluginConfig = (on, config) => {
+    config.baseUrl = 'http://localhost:9000';
+    return performCommonSetup(on, config, getWebpackConfig({}));
+};
+export default setupPluginsAndConfig;
```

Comment:
It may be a minor thing, but I think that it increases the readability 💪 

```suggestion
const setupPluginsAndConfig: Cypress.PluginConfig = (on, config) => {
    config.baseUrl = 'http://localhost:9000';
    return performCommonSetup(on, config, getWebpackConfig({}));
};

export default setupPluginsAndConfig;
```